### ISSUE TITLE
Set Play/Pause to F11

### ISSUE THREAD

**Issue by joelteon** [beardedspice/beardedspice#153]
F11 isn't mapped to anything on my computer, but I can't set Play/Pause to F11 on my external keyboard. Since I'm using F13-F15 for volume, F10-F12 fits nicely with that, so I don't want to pick a different key sequence. How can I set Play/Pause to F11?

**Comment by foozee** [comment]
Hi @joelteon, By default, F11 is set for `Show Desktop` .... to use F11 as a shortcut in BS, you'll need to change the F11 shortcut in mac:

1. open `system preferences`
1. search for `shortcuts`
1. Under `Mission Control` ... change the `Show Desktop` shortcut to anything (I tried setting it to ^ + F11) ... however, disabling it (by unchecking the checkbox) didn't seem to work :)
1. go to BS's preferences under shortcuts you can now  register `F11` for whatever :wink:

**Comment by joelteon** [comment]
Okay, but "Show Desktop" doesn't have anything bound to it on my machine and I still can't set anything to F11 in BeardedSpice.

**Comment by foozee** [comment]
does BS show any error ? or what happens exactly, how can we reproduce the issue u r having please be as detailed as possible... also, what keyboard are u using, and are u using the latest version of BS ?

**Comment by Stillness-2** [comment]
http://www.youtube.com/watch?v=pgf8QPLaSj0

**Comment by foozee** [comment]
@Stillness-2 video is private :(

**Comment by Stillness-2** [comment]
Try now :)

**Comment by joelteon** [comment]
@foozee It says "The key combination F11 cannot be used", and below that "This combination cannot be used because it is already used by a system-wide keyboard shortcut.
If you really want to use this key combination, most shortcuts can be changed in the Keyboard & Mouse panel in System Preferences."

**Comment by Stillness-2** [comment]
um "Since I'm using F13-F15 for volume" and  "Keyboard I'm using is this one":)
Not that I am picky, but I don't understand, unfortunately.

**Comment by Stillness-2** [comment]
We need more accurate information

**Comment by Stillness-2** [comment]
@joelteon, saw you the video, the link to which I gave?

**Comment by joelteon** [comment]
Yeah, that was my mistake. The keyboard I'm concerned with is my external one, not the Macbook keyboard. It has F1-F15 keys. I can't set anything to F11, F8, or F12 in BeardedSpice because BeardedSpice believes those are bound to something already. but I'm not sure how to figure out what it thinks they are bound to.

**Comment by joelteon** [comment]
![title](https://s3.amazonaws.com/f.cl.ly/items/3p38031t1f0n0V3O1U0j/screendump%202015-05-11%20at%2010.17.50%20AM.png)

**Comment by chrishough** [comment]
I am having this same issue, were you able to resolve this @joelteon
